tensorflow_addons/optimizers/weight_decay_optimizers.py

```diff
@@ -39,10 +39,11 @@ class DecoupledWeightDecayExtension:
 
     This class alone is not an optimizer but rather extends existing
     optimizers with decoupled weight decay. We explicitly define the two
-    examples used in the above paper (SGDW and AdamW), but in general this
-    can extend any OptimizerX by using
-    `extend_with_decoupled_weight_decay(
-        OptimizerX, weight_decay=weight_decay)`.
+    examples used in the above paper (SGDW and AdamW), but in general this can 
```

Comment:
Please do flake8 and black before adding commits as below.
`python -m flake8 <file_path>` to check coding style errors.
`python -m black <file_path>` to reformat the files if needed.

Reply:
As a convenience we have a git pre-commit hook you can run to validate formatting:
https://github.com/tensorflow/addons/blob/master/CONTRIBUTING.md#coding-style